### ISSUE TITLE
Request for inclusion: sklearn-xarray

### ISSUE THREAD

**Issue by phausamann** [scikit-learn-contrib/scikit-learn-contrib#37]
Request for project inclusion in scikit-learn-contrib

* Project name: sklearn-xarray
* Project description: [xarray](http://xarray.pydata.org/en/stable/) compatibility wrappers for sklearn
* Authors: Peter Hausamann
* Current repository: https://github.com/phausamann/sklearn-xarray
* Requirements:
 * [x] scikit-learn compatible (check_estimator passed)
 * [x] Documentation: https://phausamann.github.io/sklearn-xarray/ 
 * [x] Unit tests (coverage: 93%)
 * [x] Python3 compatible
 * [x] PEP8 compliant
 * [x] Continuous integration

This project makes sklearn estimators compatible with xarray DataArrays and Datasets, i.e. it provides a wrapper mechanism that keeps meta-data intact when calling the usual estimator methods. Wrapped estimators pass the `check_estimator` tests and can be used in pipelines.

It also adds a couple of new features such as allowing for estimators to change the number of samples,  label-based cross-validation and various xarray-specific pre-processing transformers.

I've been working on this project for about a year and I think that by now it is mature enough to be included in scikit-learn-contrib.
